src/components/molecules/UiScale/UiScale.vue

```diff
@@ -16,6 +16,7 @@
       >
         <legend
           v-if="legend"
+          id="scale-legend"
```

Comment:
@jbucior `id` should be unique what about when dev put 2 `UiScale` on view ;) 
IMO should be 
```
const scaleId = computed(() => (
  props.id || `scale-${uid()}`
));